Toolbox/scripts/sshd_install.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+
+SCRIPTDIR=$( cd -P -- "$(dirname -- "$(command -v -- "$_")")" && pwd -P )
+SSD_INSTALL_DIR="/net/mmx/mnt/app/eso/hmi/engdefs/scripts/ssh"
+
+export LD_LIBRARY_PATH=/net/mmx/mnt/app/root/lib-target:/net/mmx/eso/lib:/net/mmx/mnt/app/usr/lib:/net/mmx/mnt/app/armle/lib:/net/mmx/mnt/app/armle/lib/dll:/net/mmx/mnt/app/armle/usr/lib
+export PATH=${SCRIPTDIR}:${SSD_INSTALL_DIR}/usr/bin:${SSD_INSTALL_DIR}/usr/sbin:/proc/boot:/bin:/usr/bin:/usr/sbin:/sbin:/mnt/app/media/gracenote/bin:/mnt/app/armle/bin:/mnt/app/armle/sbin:/mnt/app/armle/usr/bin:/mnt/app/armle/usr/sbin
+
+. ${SCRIPTDIR}/util_mountsd.sh
+if [[ -z "$VOLUME" ]] 
+then
+  echo "No SD-card found, quitting"
+  exit 0
+fi
+
+PUB_KEY_PATH="${VOLUME}/Custom/id_rsa.pub"
+SSHD_APP="${VOLUME}/Toolbox/apps/sshd"
+
+
+# Make it writable
+mount -uw /mnt/app
+mount -uw /mnt/system
+
+mkdir -p ${SSD_INSTALL_DIR}/etc
+
+cp -prv ${SSHD_APP}/etc/* ${SSD_INSTALL_DIR}/etc/
+cp -prv ${SSHD_APP}/usr ${SSD_INSTALL_DIR}/
+chmod 755 ${SSD_INSTALL_DIR}/usr/bin/*
+chmod 755 ${SSD_INSTALL_DIR}/usr/sbin/*
+sed -ir 's:\r$::g' ${SSD_INSTALL_DIR}/usr/sbin/start_sshd
+sed -ir 's:\r$::g' ${SSD_INSTALL_DIR}/etc/banner.txt
+
+if [ -f ${PUB_KEY_PATH} ]; then
+  mkdir -p /root/.ssh
+  chmod 600 /root/.ssh
+  chmod 644 /mnt/app/root
+  cp -prv ${PUB_KEY_PATH} /root/.ssh/authorized_keys
```

Comment:
Are we overwriting existing files here? If yes can we backup them bevore so we have a clean way back?

Reply:
Nope, this file definitely doesn't already exist.